src/struct_info.json

```diff
@@ -65,6 +65,12 @@
             ]
         }
     },
+    {
+        "file": "limits.h",
+        "defines" : [
+            "PATH_MAX"
+        ]
+    },
```

Comment:
This part should no longer be needed I think?   (I try to only include the things we actually use in this file).